File: Step 1/Step 1.2/pattern5/pattern5.java
package pattern5;

import java.util.*;

public class pattern5 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int n;
        System.out.print("Enter the number of rows ");
        n = sc.nextInt();
        int i, j;
        for (i = 0; i<n; i++) {
            for (j=n; j>i; j--) {
                System.out.print("* ");
            }
            System.out.println();
        }

    }
}

File: Step 1.2/pattern5/pattern5.cpp
#include <iostream>
using namespace std;

int main()
{

    int n;
    cout << "Enter the number of rows ";
    cin >> n;
    int i, j;
    for (i = 0; i < n; i++)
    {
        for (j = n; j > i; j--)
        {
            cout << "* ";
        }
        cout << endl;
    }
}